test/integration/test_backend.py

```diff
@@ -199,3 +199,70 @@ def test_backend_deepcopy(self):
                 backend_copy._api_client._session.base_url,
                 backend._api_client._session.base_url,
             )
+
+    def test_backend_pending_jobs(self):
+        """Test pending jobs are returned."""
+        backends = self.service.backends()
+        self.assertTrue(any(backend.status().pending_jobs > 0 for backend in backends))
+
+    def test_backend_fetch_all_qubit_properties(self):
+        """Check retrieving properties of all qubits"""
+        num_qubits = self.backend.num_qubits
+        qubits = list(range(num_qubits))
+        qubit_properties = self.backend.qubit_properties(qubits)
+        self.assertEqual(len(qubit_properties), num_qubits)
+        for i in qubits:
+            self.assertIsInstance(qubit_properties[i], IBMQubitProperties)
+
+    def test_sim_backend_options(self):
+        """Test simulator backend options."""
+        backend = self.service.backend("ibmq_qasm_simulator")
+        backend.options.shots = 2048
+        backend.set_options(memory=True)
+        inputs = backend.run(ReferenceCircuits.bell(), shots=1, foo="foo").inputs
+        self.assertEqual(inputs["shots"], 1)
+        self.assertTrue(inputs["memory"])
+        self.assertEqual(inputs["foo"], "foo")
+
+    @production_only
+    def test_paused_backend_warning(self):
+        """Test that a warning is given when running jobs on a paused backend."""
+        backend = self.service.backend("ibmq_qasm_simulator")
+        paused_status = backend.status()
+        paused_status.status_msg = "internal"
+        backend.status = mock.MagicMock(return_value=paused_status)
+        with self.assertWarns(Warning):
+            backend.run(ReferenceCircuits.bell())
+
+    def test_backend_wrong_instance(self):
+        """Test that an error is raised when retrieving a backend not in the instance."""
+        backends = self.service.backends()
+        hgps = self.service._hgps.values()
+        if len(hgps) >= 2:
+            for hgp in hgps:
+                backend_names = list(hgp._backends)
+                for backend in backends:
+                    if backend.name not in backend_names:
+                        with self.assertRaises(QiskitBackendNotFoundError):
+                            self.service.backend(
+                                backend.name,
+                                instance=f"{hgp._hub}/{hgp._group}/{hgp._project}",
+                            )
+                        return
```

Comment:
All of the tests like these that use `._hgps` will have to be updated to be quantum channel only because they will not work on cloud